Design/Colors.json

```diff
@@ -20,8 +23,8 @@
   "navy_300": "EFEFF3",
   "navy_400": "DBDEE7",
   "navy_500": "95959E",
-  "navy_600": "6B7180",
-  "navy_700": "062340",
+  "navy_600": "626369",
+  "navy_700": "353535",
```

Comment:
this is more of a question for @maggieKSR and design systems, but `353535` doesnt seem very navy at all. should this not be moved into a grey value?